### ISSUE TITLE
Annotation image appears distorted 

### ISSUE THREAD

**Issue by yokotenka** [WEHI-labatlab/HE_GUI#2]
The annotation image when loaded is distorted. This is a visual effect and does not seem like it has an effect on the outcome. Needs further looking into

**Comment by yokotenka** [comment]
` self.target_image = he_script.resize_(self.source_image, self.target_image)` is called in 
[https://github.com/WEHI-labatlab/HE_GUI/blob/main/heGUI/heGUI.py](url)

Resizing the target image (with annotation) to the same size as the optical image. We can handle it differently by padding the target image if the image sizes are different.

**Comment by christianrickert** [comment]
Padding would be better IMHO. My H&E references never match the gold slide image in dimensions.
Alternatively, would it be feasible to introduce two scaling factors (X/Y) for the coordinate system of the target image? - And display the target image with its original dimensions without padding or distortion?

**Comment by yokotenka** [comment]
I think that's a good idea. So if I understand correctly, we display the images as is, but once we place all the landmarks, we "scale" the landmark coordinates.

**Comment by christianrickert** [comment]
Exactly! - The "scaling" would also apply to pairs of images with (differing) resolution information, i.e. "pixels per distance". Here's an example for such information embedded in the metadata of TIFF files:
https://www.awaresystems.be/imaging/tiff/tifftags/resolutionunit.html
